snapcraft/cli/help.py

```diff
@@ -65,8 +75,16 @@ def help_command(topic, devel):
         try:
             _module_help(topic, devel)
         except ImportError:
-            echo.error('The plugin does not exist. Run `snapcraft '
-                       'list-plugins` to see the available plugins.')
+            click.echo(dedent("""\
```

Comment:
can _module_help raise and exception that takes care of this message and formatting, so it can be in `snapcraft/internal/errors` like the rest?

Reply:
I want to keep things implemented in cli independent from `internal.errors`. That would also print the error in red which is just horrible for so much text.